client/webserver/api.go

```diff
@@ -418,6 +418,40 @@ func (s *WebServer) apiUpdateCert(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, simpleAck(), s.indent)
 }
 
+func (s *WebServer) apiUpdateDEXHost(w http.ResponseWriter, r *http.Request) {
+	form := new(struct {
```

Comment:
All the other literals in this file are defined with `&struct {` rather than `new(struct {`, so just for the sake of consistency lets use `&` here too.